tools/bootstrap_tests.py

```diff
@@ -148,6 +148,6 @@ def create_influxdb_database():
     if os.getenv('POLICY') == 'belleii':
         belleii_bootstrap(client)
 
-    response = create_influxdb_database()
-    if response.status_code != 201:
-        print('Failed to create rucio database in influxDB : %s' % response.text)
+    #response = create_influxdb_database()
```

Comment:
I see what you did here :)

Reply:
Creative problems require creative solutions :-)